holmes/core/tool_calling_llm.py

```
@@ -290,6 +228,125 @@ def __init__(
+                    # Execute the tool
+                    llm_tool_result = self._invoke_llm_tool_call(
+                        tool_to_call=mock_tool_call,
+                        previous_tool_calls=[],
+                        trace_span=DummySpan(),
+                        tool_number=None,
+                    )
+
+                    # Add tool result to messages
+                    messages.append(llm_tool_result.as_tool_call_message())
+
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)